lib/util/stream_utils.js

```diff
@@ -473,6 +475,9 @@ shaka.util.StreamUtils = class {
     if (textTrack.kind) {
       track.roles = [textTrack.kind];
     }
+    if (textTrack.kind == 'forced') {
+      track.forced = true;
```

Comment:
Like above, is this Apple-specific?  I don't see anything in the HTML spec for this.  Do you have any documentation you could link to, or is this just from trying it?

Reply:
This is the way Safari works. I have seen it testing with https://devimages.apple.com.edgekey.net/streaming/examples/bipbop_16x9/bipbop_16x9_variant.m3u8 and other private streams.